.github/ISSUE_TEMPLATE/docs.yml

```diff
@@ -0,0 +1,46 @@
+name: Documentation
+description: Are the docs lacking or missing something?
+labels: ["documentation :book:"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > Note: This regards the [GitHub Pages Documentation](https://giscience.github.io/openrouteservice/).
```

Comment:
Since the playground isn't publicly available, this is probably the issue that people would use for playground problems. This should be added here as well.

Reply:
Done in a3447b0c1d7448367ab5add79b62103b4d53d4fd